site/en/blog/blinkng/index.md

```diff
@@ -175,10 +175,10 @@ To understand why this is important, and where else it may lead, we need to cons
+Now that BlinkNG is approaching completion, we are already starting to explore this area; [Non-Blocking Commit](http://crbug.com/1255972) is a first foray into changing the renderer's threading model. _Compositor commit_ (or just _commit_) is a synchronization step between the main thread and the compositor thread. During commit, we make copies of rendering data that are produced on the main thread, to be used by the downstream compositing code running on the compositor thread. While this synchronization is happening, main thread execution is stopped while the copying code runs on the compositor thread. This is done to ensure that the main thread doesn't modify its rendering data while the compositor thread is copying it.
 
 Non-Blocking Commit will eliminate the need for the main thread to stop and wait for the commit stage to end—the main thread will continue doing work while commit runs concurrently on the compositor thread. The net effect of Non-Blocking Commit will be a reduction in the time dedicated to rendering work on the main thread, which will decrease congestion on the main thread, and improve performance. As of this writing (March 2022), we have a working prototype of Non-Blocking Commit, and we're preparing to do a detailed analysis of its impact on performance.
 
 Waiting in the wings is [Off-main-thread Compositing](https://docs.google.com/document/d/1BfIqMcJvmMFdk8e_KKeOEofz16610Mql6gmfeFhrsZk/preview), with the goal of making the rendering engine match the illustration by moving _layerization_ off the main thread, and onto a worker thread. Like Non-Blocking Commit, this will reduce congestion on the main thread by diminishing its rendering workload. A project like this would never have been possible without the architectural improvements of Composite After Paint.
 
-And there are more projects in the pipeline (pun intended)! We finally have a foundation that makes it possible to experiment with redistributing rendering work, and we are very excited to see what's possible!
\ No newline at end of file
+And there are more projects in the pipeline (pun intended)! We finally have a foundation that makes it possible to experiment with redistributing rendering work, and we are very excited to see what's possible!
```

Comment:
Is this an end-of-file newline diff? I think this was previously a problem.

Reply:
Everything else lgtm.

Reply:
Looks fine to me in preview, is this a concern @rachelandrew?

Reply:
I think it's fine, if the preview is building it should be.